ParkingZoneApp/Areas/Admin/Controllers/ParkingSlotController.cs

```diff
@@ -61,4 +63,50 @@ public IActionResult Create(CreateViewModel createModel)
         return View(createModel);
     }
     #endregion
+
+    #region Edit
+    // GET: Admin/ParkingSlot/Edit/5
+    public IActionResult Edit(int? id)
+    {
+        var slot = _slotService.GetById(id);
+        if (slot == null)
+        {
+            return NotFound();
+        }
+        var editModel = new EditViewModel(slot);
+        return View(editModel);
+    }
+
+    [HttpPost]
+    [ValidateAntiForgeryToken]
+    public IActionResult Edit(int id, EditViewModel editModel)
+    {
+        if (id != editModel.Id)
+        {
+            return NotFound();
+        }
+
+        if (ModelState.IsValid)
+        {
+            try
+            {
+                ParkingSlot slot = editModel.MapToModel(editModel);
+                _slotService.Update(slot);
```

Comment:
firstly you are updating slot and then checking this is slot exists with this number ?!

Reply:
Well, I have written it by looking at parkingzone controller which was added scaffold. And it has not caused any problem yet.